tests/rados/test_bluestore_min_alloc_size.py

```diff
@@ -134,6 +133,71 @@ def run(ceph_cluster, **kw):
                     f"OSDs successfully deployed with the new alloc size, and verified the size on OSD: {osd_id}"
                 )
 
+                """
+                enhance the tests by adding ceph config show osd.id and ceph
+                daemon osd.id config show
+                """
+                # Enhanced test: Check ceph config show and daemon output for each OSD
+                show_config_hdd = int(
+                    mon_obj.show_config(
+                        daemon="osd", id=osd_id, param="bluestore_min_alloc_size_hdd"
+                    )
+                )
+
+                show_config_ssd = int(
+                    mon_obj.show_config(
+                        daemon="osd", id=osd_id, param="bluestore_min_alloc_size_ssd"
+                    )
+                )
+
+                print(
+                    "The value of show_config_hdd:",
+                    show_config_hdd,
+                    type(show_config_hdd),
+                )
+                print(
+                    "The value of show_config_ssd:",
+                    show_config_ssd,
+                    type(show_config_ssd),
+                )
+                print(
+                    "The value of custom_min_alloc_size:",
+                    custom_min_alloc_size,
+                    type(custom_min_alloc_size),
+                )
```

Comment:
Please remove these lines if no longer needed, if we want to retain them, please use log.info or log.debug functionality

Reply:
Issue resolved, I removed the lines, I added those lines just to know which type it's returning.